src/pyggdrasil/tree_inference/_huntress.py

```diff
@@ -0,0 +1,58 @@
+"""Tree inference with the HUNTRESS algorithm."""
+
+import anytree
+import numpy as np
+import pandas as pd
+import scphylo
+import pyggdrasil._scphylo_utils as utils
+
+
+def huntress_tree_inference(
+    mutations: np.ndarray,
```

Comment:
Good points!

My understanding is that HUNTRESS:
  - doesn't handle homozygous entries,
  - I'm unsure if it can handle missing entries,
so for now we assume that the entries are 0 or 1. (As said in docstring).


Regarding the shape, we need to make sure we have one convention and we stick to it (also documenting it or annotating the shapes with `jaxtyping`). For example, here the docstring suggests that it's `(cells, sites)`, which indeed is the transpose of the convention you mentioned:  
```
def attach_cells_to_tree(
    rng: interface.JAXRandomKey,
    tree: interface.TreeAdjacencyMatrix,
    n_cells: int,
    strategy: CellAttachmentStrategy,
) -> PerfectMutationMatrix:
    """Attaches cells to the mutation tree.

    Args:
        rng: JAX random key
        tree: matrix representing mutation tree
            with the highest index representing the root
        n_cells: number of cells to sample
        strategy: cell attachment strategy.
          See ``CellAttachmentStrategy`` for more information.

    Returns:
        binary matrix of shape ``(n_cells, n_sites)``,
          where ``n_sites`` is determined from the ``tree``
          NOTE: Last row will be all ones is the root node.
          NOTE: not truncated the last row as shown in the SCITE paper
    """
    ...
```

Reply:
I opened #31 to address this problem. We can choose the convention at the meeting and include this in the next sprint.